tests/integration_tests/performance/test_snapshot_restore_performance.py

```diff
@@ -186,94 +186,35 @@ def consume_output(cons, result):
 
 
 @pytest.mark.nonci
-@pytest.mark.parametrize("vcpu_count", [1, 2, 4, 8, 10])
-def test_snapshot_scaling_vcpus(
-    bin_cloner_path, microvm_factory, rootfs, guest_kernel, vcpu_count, st_core
+@pytest.mark.parametrize(
+    "mem, vcpus",
+    [
+        (128, 1),
+        (1024, 1),
+        (2048, 2),
+        (4096, 3),
+        (6144, 4),
+        (8192, 5),
+        (10240, 6),
+        (12288, 7),
+    ],
+)
+def test_snapshot_scaling(
+    bin_cloner_path, microvm_factory, rootfs, guest_kernel, st_core, mem, vcpus
 ):
-    """Restore snapshots with variable vcpu count."""
-    guest_config = f"{vcpu_count}vcpu_{BASE_MEM_SIZE_MIB}mb"
-    env_id = f"{guest_kernel.name()}/{rootfs.name()}/{guest_config}"
-    st_prod = st.producer.LambdaProducer(
-        func=get_snap_restore_latency,
-        func_kwargs={
-            "vm_builder": MicrovmBuilder(bin_cloner_path),
-            "microvm_factory": microvm_factory,
-            "guest_kernel": guest_kernel,
-            "rootfs": rootfs,
-            "vcpus": vcpu_count,
-            "mem_size": BASE_MEM_SIZE_MIB,
-        },
-    )
-    st_cons = default_lambda_consumer(env_id, WORKLOAD)
-    st_core.add_pipe(st_prod, st_cons, f"{env_id}/{WORKLOAD}")
-    st_core.name = TEST_ID
-    st_core.custom["guest_config"] = guest_config
-    st_core.run_exercise()
+    """
+    Restores snapshots with vcpu/memory configuration, roughly scaling according to mem = vcpus * 2048MB,
```

Comment:
The set up looks not obeying this equation. It seems to be (vcpus - 1) * 2048?

Reply:
oh, you're right!